modules/gsp-cluster/vpc-flow-logs.tf

```diff
@@ -0,0 +1,67 @@
+// TODO add variables somewhere
+
+resource "aws_flow_log" "vpc_flow_log" {
+  iam_role_arn         = "${var.cluster_name}_cloudwatch_vpc_flow_log_shipping_role"
+  log_destination      = "${aws_cloudwatch_log_group.vpc_flow_logs.arn}"
+  log_destination_type = "cloud-watch-logs"
+  traffic_type         = "ALL"
+  vpc_id               = "${var.vpc_id}"
+}
+
+
+resource "aws_cloudwatch_log_group" "vpc_flow_logs" {
+  name              = "${var.cluster_domain}_vpc_flow_log"
+  retention_in_days = 30
```

Comment:
Assuming this will all end up in splunk, are CyberSec/IA happy with this retention period?